runtime/src/lib.rs

```diff
@@ -861,12 +865,26 @@ construct_runtime!(
         UncheckedExtrinsic = UncheckedExtrinsic
     {
         System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Call, Storage},
         Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
-        //Indices: pallet_indices::{Pallet, Call, Storage, Event<T>, Config<T>},
-        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-        TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
         Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
+        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage},
+        TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
+
+        // PARACHAIN
+		ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Config, Storage, Inherent, Event<T>, ValidateUnsigned},
+        ParachainInfo: parachain_info::{Pallet, Storage, Config},
+
+        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
+
+        // Consensus
+        Aura: pallet_aura::{Pallet, Config<T>},
+        AuraExt: cumulus_pallet_aura_ext::{Pallet, Config},
+
+        XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>},
+        PolkadotXcm: pallet_xcm::{Pallet, Call, Event<T>, Origin},
+        XcmHandler: cumulus_pallet_xcm::{Pallet, Call, Event<T>, Origin},
+        DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>},
+
         GeneralCouncil: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>},
```

Comment:
rename like in standalone chain?
add other runtime modules like in standalone chain?

Reply:
We can do this as a part of https://github.com/DataHighway-DHX/node/issues/227